quran/urls.py

```diff
@@ -0,0 +1,17 @@
+'''Quran app api url endpoints'''
+from django.urls import re_path
+
+from . import views
+
+urlpatterns = [
+    re_path(
+        r'quran/(?P<surah_id>\d+)/$',
```

Comment:
Why do you use regular expressions? I think it's an old style ;)

Reply:
`quran`  that's GREAT book, but bad name for our situation. Please rename it or add /surah/
so it's more readable when we have `quran/surah/` or just `surah`